app/elements/pages/page-browse.html

```diff
@@ -335,9 +310,6 @@ <h2 class="flex"><span>[[pageTitle]]</span> <span class="count">[[_elementCount]
       _actualView: function(view, force) {
         return force ? 'cards' : view;
       },
-      _updateForceCards: function() {
-        this._forceCards = this._narrowViewport || this._detectIEVersion() > 0;
-      },
```

Comment:
Let's also remove the whole `_detectIEVersion` function https://github.com/Polymer/polymer-element-catalog/blob/layout/app/elements/pages/page-browse.html#L290